src/core/body/body.view.d.ts

```diff
@@ -0,0 +1,21 @@
+import {View} from '../view/view';
```

Comment:
there are no type definition for most of the files, they will be added as soon as possible)